analysis/src/vm/batcher.rs

```diff
@@ -154,7 +154,10 @@ mod tests {
     use crate::vm::test_utils::batch_str;
 
     fn test_batching(path: &str) {
-        let base_path = PathBuf::from("../test_data/asm/batching");
+        let base_path = PathBuf::from(format!(
+            "{}/../test_data/asm/batching",
+            env!("CARGO_MANIFEST_DIR")
+        ));
```

Comment:
```suggestion
        let base_path = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("../test_data/asm/batching");
```

Think this is cleaner?